pkg/apiserver/registry/intelligence/networkpolicyrecommendation/rest.go

```diff
@@ -60,6 +69,15 @@ func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions)
 	}
 	intelliNPR := new(intelligence.NetworkPolicyRecommendation)
 	r.copyNetworkPolicyRecommendation(intelliNPR, npReco)
+	// Try to retrieve result from ClickHouse in case NPR is completed
+	if npReco.Status.State == crdv1alpha1.NPRecommendationStateCompleted {
```

Comment:
Do you think we need to add this to List() as well?

Reply:
It seems we only display `CreationTime, CompletionTime, Name, Status` for the list command in CLI. Maybe we can skip result retrieving in the list API to save time?

Reply:
Since get() and list() are similar, will it be better to have consistency for these two functions? I don't have strong opinion. As you said, we only display these 4 columns when using list command.

Reply:
Got it. Yes, it makes more sense to keep the consistency considering the generality of the API. I updated the list function. Thanks!